lms/djangoapps/courseware/module_render.py

```diff
@@ -713,7 +713,6 @@ def rebind_noauth_module_to_user(module, real_user):
         wrappers=block_wrappers,
         get_real_user=user_by_anonymous_id,
         services={
-            'i18n': ModuleI18nService(),
             'fs': FSService(),
             'field-data': field_data,
             'user': DjangoXBlockUserService(user, user_is_staff=user_is_staff),
```

Comment:
@asadiqbal08 -- I caught up with @cpennington today to understand more about why this particular line is necessary.  It seems there is an LTI use case we need to support, which has to do with when the LTI XBlock is loaded.  The XBlock needs to re-bind to the user associated with the authentication information passed along with the XBlock handler contents.  The good news is that the fix should be as easy as restoring the above line.

In addition to reverting the above change, however, we should ensure that the LTI use case is not impacted in the future by protecting it with a unit test.  The test can utilize a simple XBlock that calls the re-bind method and confirms the proper services are both defined and the same before and after the re-binding process is complete.

@cpennington, if there's any more context/information to add here so @asadiqbal08 can write up the proper test, please feel free to share -- thanks!